web/styles/subscriptions.css

```diff
@@ -218,6 +218,10 @@ h4.user_group_setting_subsection_title {
     padding-right: 8px;
 }
 
+.subscriber-name a i {
```

Comment:
Let's avoid element selectors like this; it's okay to be verbose and list multiple classes as needed, rather than putting generics like this. Consider also whether that `a` descendant selector is needed.